src/components/account/account-dropdown.vue

```diff
@@ -0,0 +1,82 @@
+<!--
+  - Copyright (c) TIKI Inc.
+  - MIT license. See LICENSE file in root directory.
+  -->
+
+<script setup lang="ts">
+import { ref, watch } from "vue";
+import { icon } from "@/service/receipt/receipt-account-type";
+const emits = defineEmits(["input"]);
+const props = defineProps({
+  accounts: {
+    type: Array,
+    required: true,
+  },
+});
+
+const selectedAccount = ref<HTMLSelectElement>();
+
+watch(selectedAccount, () => {
+  emits("input", selectedAccount.value);
+});
+</script>
+
+<template>
+  <v-select
+    :options="accounts"
+    class="account-select"
+    v-model="selectedAccount"
+  >
+    <template #option="option" v-slot:item="option">
+      <img :alt="option.label" :src="icon(option.label)" style="width: 40px" />
+      <span class="option-label">{{ option.label }}</span>
+    </template>
+  </v-select>
+</template>
+
+<style>
+.account-select .vs__dropdown-toggle {
+  border: none;
+}
+.account-select .vs__dropdown-menu {
+  border-radius: 0.5em;
+  padding: 5px 8px;
+}
+.account-select .vs__open-indicator {
+  padding: 10px 30px 10px 10px;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjUiIGhlaWdodD0iMTUiIHZpZXdCb3g9IjAgMCAyNSAxNSIgZmlsbD0ibm9uZSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj4KPHBhdGggZD0iTTEuMTgxODIgMi4wMTMxOEwxMi4yMzY0IDEyLjAxMzJMMjMgMi4wMTMxOCIgc3Ryb2tlPSJibGFjayIgc3Ryb2tlLW9wYWNpdHk9IjAuNiIgc3Ryb2tlLXdpZHRoPSIzIi8+Cjwvc3ZnPgo=");
```

Comment:
Use SVG assets in the project, not remote URLs

Reply:
ok! fixed!